sycl/doc/extensions/supported/sycl_ext_intel_device_info.md

```diff
@@ -46,7 +46,7 @@ A new aspect, ext\_intel\_device\_id, will be added.
 
 ## Error Condition ##
 
-An invalid object runtime error will be thrown if the device does not support aspect\:\:ext\_intel\_device\_id.
+An exception with a feature not supported error code will be thrown if the device does not support aspect\:\:ext\_intel\_device\_id.
```

Comment:
```suggestion
Throws a synchronous `exception` with the `errc::feature_not_supported` error
code if the device does not have `aspect::ext_intel_device_id`.
```

We use this same type of wording in the core SYCL spec to describe the exceptions that are thrown, so let's use that wording here too.